components/avatars/avatars.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <script type="text/javascript" src="https://livejs.com/live.js"></script>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="icon"
+    href="data:image/svg+xml, %3Csvg width='120' height='120' viewBox='0 0 60 72' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='%23c1f79a' d='M42.334 49.147a29.945 29.945 0 0 1-19.338-8.151c-8.014-7.365-8.378-18.076-8.533-22.649l-.022-.627a2.904 2.904 0 0 1 3.457-2.951c17.005 3.355 21.695 16.324 22.056 17.4a49.543 49.543 0 0 1 3.574 15.922 1 1 0 0 1-.967 1.052c-.029.001-.106.004-.227.004Z'/%3E%3Cpath fill='%2345C14E' d='M44.435 55.316c-11.645 0-17.375-6.974-17.652-7.354a1 1 0 0 1 .262-1.424 11.103 11.103 0 0 1 12.774-1.574c-1.465-9.078 1.877-13.568 2.031-13.77a.998.998 0 0 1 .75-.39.97.97 0 0 1 .78.325c8.944 9.771 8.793 16.532 7.908 19.691-.034.14-1.062 4.092-4.772 4.406-.711.062-1.405.09-2.08.09Z'/%3E%3Cg fill='none' stroke='%23000' stroke-linecap='round' stroke-linejoin='round' stroke-width='2'%3E%3Cpath d='M55.184 57.69S34.96 45.877 23.097 24.206m22.131 30.096c-11.93.46-17.628-6.88-17.628-6.88'/%3E%3Cpath d='M40.528 42.483c-.56-7.195 2.116-10.679 2.116-10.679 8.834 9.654 8.406 16.162 7.681 18.747m-13.311-3.129a30.15 30.15 0 0 1-13.341-7.162c-8.072-7.419-8.067-18.241-8.232-22.577a1.903 1.903 0 0 1 2.264-1.932C34.694 19.103 39.02 32.528 39.02 32.528'/%3E%3C/g%3E%3C/svg%3E%0A">
+  <title>Feather UI</title>
+  <link rel="stylesheet" href="/css/main.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;300;400;500;700&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/7b62ea7364.js" crossorigin="anonymous"></script>
+  <script src="/js/_header/header.js" type="text/javascript" defer></script>
+  <script src="/js/_footer/footer.js" type="text/javascript" defer></script>
+  <script src="/js/_sideNav/sideNav.js" type="text/javascript" defer></script>
+  <script src="/js/_topSideNav/topSideNav.js" type="text/javascript" defer></script>
+</head>
+
+<body>
+  <header-component></header-component>
```

Comment:
Is the header-component required here ?

Reply:
Yes. Otherwise, the header will not be visible on the avatar page or any other component page for that matter.